acq4/devices/PatchPipette/states.py

```diff
@@ -158,7 +158,6 @@ def initializeClamp(self):
                 cdev.setHolding(mode=mode, value=holding)
             if tpParams is None:
                 tpParams = {}
-            tpParams.setdefault('clampMode', mode)
```

Comment:
This is the only bit of this PR I can't easily justify. Why don't we want to set the `clampMode` on the test pulse params, @campagnola ?

Reply:
Initial clamp mode should be set only once when the state begins; the test pulse then uses whatever mode is currently in use. (otherwise the test pulse is constantly enforcing a particular clamp mode, which we usually don't want)